src/Tasks.UnitTests/RoslynCodeTaskFactory_Tests.cs

```diff
@@ -969,6 +1027,47 @@ public void SourceCodeFromFile()
             }
         }
 
+        [Fact]
+        public void MismatchedTaskNameAndTaskClassName()
+        {
+            const string taskName = "SayHello";
+            const string className = "HelloWorld";
+            taskName.ShouldNotBe(className, "The test is misconfigured.");
+            const string errorMessage = $@"The task name ""{taskName}"" could not be found.";
```

Comment:
To be future proof - can you reference the `"CodeTaskFactory.CouldNotFindTaskInAssembly"` resource string here?

Reply:
Done